src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/JS/WasmTestCommand.cs

```diff
@@ -67,33 +69,51 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
                 engineBinary = FindEngineInPath(engineBinary + ".cmd");
 
-            var engineArgs = new List<string>();
-
-            if (_arguments.Engine == JavaScriptEngine.V8)
+            var cts = new CancellationTokenSource();
+            try
             {
-                // v8 needs this flag to enable WASM support
-                engineArgs.Add("--expose_wasm");
-            }
+                var serverURLs = await WebServer.Start(
+                    _arguments, logger,
+                    null,
+                    cts.Token);
```

Comment:
Do we need to always start the web server for `wasm test` case? Can we limit it to only when `WebServerMiddlewarePaths` is set?